### DISCUSSION TITLE
License question about the meaning of EPL-2.0 OR GPL-2.0-or-later

### DISCUSSION THREAD

**Discussion by a-jp** [coin-or/CppAD#196] [Q&A]
Hi,

In the license section it says: 

> EPL-2.0 OR GPL-2.0-or-later

Can you confirm which actions, or build options etc, would enforce GPL 2.0 over EPL 2.0 for CppAD?  I would like to use EPL 2.0. 
Thanks

**Comment by bradbell** [comment]
You can use CppAD with either the EPL-2.0 license or the GPL-2.0 or later license. The choice is up to you. This part of how EPL-2.0 allows for a secondary license.

**Comment by a-jp** [comment]
Ok thanks

**Comment by bradbell** [comment]
For a more complete explanation; see
[2.3. Is EPL-2.0 with the Secondary License clause considered dual licensing?](https://www.eclipse.org/legal/epl-2.0/faq.php#h.lgjcpvoq08a9)
